### PR TITLE
Added Code for the Caw Functions (Stream and GetHashtags)

### PR DESCRIPTION
**Pull request by floan**
Tests to Follow!

### CODE REVIEW

File: src/caw/caw_function.cc [hunk 1]
```diff
@@ -182,9 +209,25 @@ void CawFunction::ReadReplys(const std::string& caw_id, KeyValueInterface& kv,
...
-  return {};
+std::vector<std::string> CawFunction::GetHashtags(const std::string& message) {
+  std::vector<std::string> hashtags; 
+  std::string hashtag = ""; 
+  int i=0; 
+  while (i < message.size()) {
+    if (message[i] == '#') {
+      i++; 
+      while (i < message.size() && std::isalnum(message[i])) {
+        hashtag += message[i++];
```
Comment (wsuppiger) [inline src/caw/caw_function.cc:220]: be careful with this since it can be a little confusing to read with multiple operations in one line

**Review by wsuppiger — APPROVED** [review]
Great work!

File: src/caw/caw_function.cc (hunk 1 above)
Comment (floan) [reply src/caw/caw_function.cc:220]: Yup definitely. Hopefully the test cases cover the edge cases!
